apps/comps/components/register-agent-block.tsx

```diff
@@ -1,18 +1,113 @@
 "use client";
 
+import { zodResolver } from "@hookform/resolvers/zod";
 import Image from "next/image";
 import React from "react";
+import { useForm } from "react-hook-form";
+import { z } from "zod";
 
 import { Button } from "@recallnet/ui2/components/shadcn/button";
+import { Card } from "@recallnet/ui2/components/shadcn/card";
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormMessage,
+} from "@recallnet/ui2/components/shadcn/form";
+import { Input } from "@recallnet/ui2/components/shadcn/input";
+
+const formSchema = z.object({
+  email: z.string().email(),
+});
+
+type FormData = z.infer<typeof formSchema>;
 
 export const RegisterAgentBlock: React.FC = () => {
+  const form = useForm<FormData>({
+    resolver: zodResolver(formSchema),
+    defaultValues: {
+      email: "",
+    },
+  });
+
+  const onSubmit = () => {};
+
   return (
-    <div className="flex h-20 w-full items-center justify-between border border-gray-600 px-10 py-12">
-      <Image src={"/agent-image.png"} alt="avatar" width={40} height={40} />
-      <span className="text-sm">REGISTER YOUR OWN AGENT. WIN REWARDS</span>
-      <Button className="bg-sky-700 px-8 py-6 text-white hover:bg-sky-600">
-        ADD AGENT
-      </Button>
+    <div className="relative left-1/2 right-1/2 ml-[-50vw] mr-[-50vw] w-screen bg-gray-900">
+      <div className="px-45 flex w-full items-center justify-around py-12">
+        <Card
+          corner="bottom-left"
+          cropSize={50}
+          className="pb-15 relative flex h-[300px] w-[300px] flex-col justify-between bg-black px-10 pt-10 xl:w-[500px]"
+        >
+          <Image
+            src="/default_agent_2.png"
+            alt="agent"
+            className="pointer-events-none absolute bottom-[-50px] right-[-80px] z-0 object-contain"
+            width={350}
+            height={350}
+          />
+
+          <h2 className="text-3xl font-semibold text-white">Add your agent</h2>
+          <div className="flex flex-col justify-between">
+            <span className="w-1/2 text-gray-400">
+              Register your own agent, win rewards
+            </span>
+            <Button className="mt-5 w-1/3 bg-white px-8 py-6 text-black hover:bg-gray-200">
+              ADD AGENT
+            </Button>
+          </div>
+        </Card>
+        <Card
+          corner="bottom-left"
+          cropSize={50}
+          className="flex h-[300px] w-[300px] items-center justify-center bg-gray-600 xl:w-[500px]"
+        >
+          <Card
+            corner="bottom-left"
+            cropSize={50}
+            className="pb-15 relative flex h-80 h-[298px] w-[298px] w-full flex-col justify-between bg-gray-900 px-10 pt-10 xl:w-[498px]"
+          >
+            <h2 className="text-3xl font-semibold text-white">
+              Never miss a competition
```

Comment:
the `Never miss a competition` card should be moved a separate component. It also replaces the `newsletter-section` component we already have